package.json

```diff
@@ -27,9 +27,9 @@
     "@sentry/node": "^5.9.0"
   },
   "devDependencies": {
-    "jest": "^23.2.0",
+    "jest": "^24.9.0",
```

Comment:
Added from Git commit ID 4f9de482a6ffe8aa210e75f8f2096a4d3f387177, because it was rebased when merged the [merge request number 5](https://github.com/MadeByThePinsHub/RecapTime-ProbotApp/pull/5)